src/js/components/Menu.js

```diff
@@ -39,6 +39,18 @@ var MenuDrop = React.createClass({
       down: this._onDownKeyPress
     };
     this.startListeningToKeyboard(this._keyboardHandlers);
+    var menuItems = this.refs.navContainer.getDOMNode().childNodes;
+    for (var i = 0; i < menuItems.length; i++) {
+      menuItems[i].setAttribute('role', 'menuitem');
```

Comment:
Is this role appropriate for nested Menu items? For things like headers?
I wonder if we should be more selective in what we place this role attribute on.